RELEASE/scripts/autoscend/paths/i_love_u_hate.ash

```diff
@@ -0,0 +1,46 @@
+boolean in_iluh()
+{
+	return my_path() == $path[11 Things I Hate About U];
+}
+
+boolean iluh_foodConsumable(string str)
+{
+	if(!in_iluh())
+	{
+		return true;
+	}
+
+	//can't consume anything with a u in it. Must have an i in it
+	if(contains_text(str, "u"))
```

Comment:
minor improvement. Can do `if(contains_text(str.to_lower_case(),"u"))` and then remove the if for "U". Same for I's